src/style.css

```
@@ -1 +1,67 @@
-/* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.navbar {
+  height: 60px;
+  display: flex;
+  flex-direction: row;
+  font-size: 12px;
+  text-transform: uppercase;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+.list__item {
+  display: block;
+  line-height: 60px;
+  margin-right: 20px;
+}
+
+.list__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  display: block;
+  text-decoration: none;
+  color: black;
```

it would be better to use hex
```suggestion
  color: #000000;
```